package.json

```diff
@@ -10,9 +10,16 @@
     "@types/node": "^16.11.38",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "emailjs-com": "^3.2.0",
```

Comment:
That's cool. I didn't know about this service. You might think about migrating to `@emailjs/browser` as the current package has become deprecated

Reply:
good catch